frontend/src/main/components/Commons/FarmStats.js

```diff
@@ -1,25 +1,31 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
 import React from "react";
 import { Card } from "react-bootstrap";
+import Cash from "./../../../assets/Cash.png";
+import Health from "./../../../assets/Health.png";
 
 const FarmStats = ({userCommons}) => {
    
     return (
         <Card>
-        <Card.Header as="h5">Your Farm Stats</Card.Header>
+        <Card.Header as="h5"><strong>Your Farm Stats</strong></Card.Header>
         <Card.Body>
             {/* update total wealth and cow health with data from fixture */}
             <Card.Text>
-                [ADD PIC HERE] 
+                <img src={Cash} alt="Cash" width="150" height="150"></img>
```

Comment:
Perhaps you could specify a CSS class here (e.g., `.icon`) to specify the image width and height.